src/status_im/ui/screens/communities/communities_list_redesign.cljs

```diff
@@ -28,7 +28,7 @@
 (def view-type   (reagent/atom  :card-view))
 (def sort-list-by (reagent/atom :name))
 
-(def community-item-data
+(def mock-community-item-data ;; TO Do remove once communities are loaded with this data.
```

Comment:
Same as my other comment about prefixing it as `TODO: ...`. It's also an old Lisp convention to use only one comma for margin comments. As a reference: https://github.com/bbatsov/clojure-style-guide#margin-inline-comments